config.yaml

```diff
@@ -20,3 +20,16 @@ options:
     default: true
     description: |
       NAT outgoing traffic
+  cidr:
+    type: string
+    default: "192.168.0.0/16"
+    description: |
+      Network CIDR assigned to Calico. This is applied to the default Calico
+      pool, and is also communicated to the Kubernetes charms for use in
+      kube-proxy configuration.
+  manage-pools:
+    type: boolean
+    default: true
+    description: |
+      If true, a default pool is created using the cidr and ipip charm
+      configuration values. Any unrecognized pools are deleted.
```

Comment:
Should we call out the unrecognized pools things more?

Reply:
How?

Reply:
“DANGER: when manage-pools is enabled, it WILL delete any pools that are unrecognized!”? I’m not entirely sure the best way, but just something to call it out well to prevent surprises.

Reply:
Updated with a lower-key variant of this.